ToDoAppWithCoreData/Controllers/ListViewController.swift

```diff
@@ -0,0 +1,178 @@
+
+import UIKit
+import CoreData
+
+protocol ListViewControllerDelegate {
+    func tapSaveButton(item: String, detail: String, beginDate: Date, beginTime: Date, endDate: Date, endTime: Date, exitingItem: NSManagedObject?)
+}
+
+
+class ListViewController: UIViewController, UITextViewDelegate {
+    
+    public var item: String = ""
```

Comment:
publicにする必要がないものはprivateの方が良いです
```suggestion
    private var item: String = ""
```